lib/index.js

```diff
@@ -24,6 +24,15 @@ class GoodLoggly extends Stream.Writable {
       useTagHeader: false,
       tags: Array.isArray(config.tags) ? config.tags : []
     });
+
+    this._buffer = [];
+
+    // Send last messages on your way out
+    this.once('finish', () => {
+      if (this._buffer.length) {
+        this._sendMessages(() => {});
```

Comment:
You could make the callback to `_sendMessages()` optional so this would be unnecessary.